### ISSUE TITLE
http.client header debug output format

### ISSUE THREAD

**Issue by 5eb188e4-63d8-44e1-abf0-6fff5707bfc5** [python/cpython#61435]
BPO | [17233](https://bugs.python.org/issue17233)
--- | :---
Nosy | @terryjreedy, @berkerpeksag, @poleto
Files | <li>[httpclient.debuglevel.txt](https://bugs.python.org/file29120/httpclient.debuglevel.txt "Uploaded as text/plain at 2013-02-19.06:52:54 by Kim.Gräsman"): Example of httpclient log output</li><li>[issue17233.patch](https://bugs.python.org/file42528/issue17233.patch "Uploaded as text/plain at 2016-04-20.04:51:46 by @poleto"): Proposed patch</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2013-02-19.06:52:54.641>
labels = ['type-bug', 'library']
title = 'http.client header debug output format'
updated_at = <Date 2019-07-29.11:51:41.347>
user = 'https://bugs.python.org/KimGrsman'
```

bugs.python.org fields:
```python
activity = <Date 2019-07-29.11:51:41.347>
actor = 'vstinner'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2013-02-19.06:52:54.641>
creator = 'Kim.Gr\xc3\xa4sman'
dependencies = []
files = ['29120', '42528']
hgrepos = []
issue_num = 17233
keywords = ['patch']
message_count = 5.0
messages = ['182367', '182689', '182694', '263792', '264057']
nosy_count = 4.0
nosy_names = ['terry.reedy', 'berker.peksag', 'Kim.Gr\xc3\xa4sman', 'luiz.poleto']
pr_nums = []
priority = 'normal'
resolution = None
stage = 'needs patch'
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue17233'
versions = ['Python 3.5', 'Python 3.6']
```

</p></details>

**Comment by crazymerlyn** [comment]
Looks like this was fixed by #6611

**Comment by picnixz** [comment]
Thank you for digging out this 10y+ issue :') It has indeed been completed in #6611 and #12681.
